test/scip/testdata/type_change.snapshot.rb

```diff
@@ -181,12 +165,12 @@ def change_type(b)
 #      | @@g (Integer(1))
 #      | ```
        @k = 1
-#      ^^ reference (write) [..] C#`@k`.
+#      ^^ reference (write) [..] `<Class:C>`#`@k`.
```

Comment:
Pretty sure this (and similar Class:D modifications below) are incorrect. There are two different things going on here -- there is a class instance var `@k` and there is an instance var `@k`. We seem to be getting confused between the two. 😦